src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<blockquote>Before continuing it is important to know that the inputs and outputs in the signature of this function are similar to dictionaries constructed based on the names, units and shapes defined in the setup.</blockquote><p>Looks a bit confuse to me.</p><p>--&gt; "Before continuing it is important to know that the <code>inputs</code> and <code>outputs</code> arguments are similar to dictionaries. Dictionary keys are variable names and values are..." etc...</p><p>(also, in the next sentences, please use "dictionary" instead of "dictionnary" )</p><p> </p><blockquote><a href="http://localhost:8888/files/02_explanation/modules/OpenMDAO/geometry?_xsrf=2%7C49769060%7Cd33ba740966d79fb41ccbed6ed276c52%7C1650526294" target="_blank">The geometry folder</a>&nbsp;contains:</blockquote><blockquote>...</blockquote><blockquote><br></blockquote><p>Here also, links to folders does not work. Please link to the group implementation. (then maybe write this list after explaining groups)</p><p> </p><blockquote>The reader will notice that the computation of the OWE is incomplete and is left as a practical exercice.</blockquote><p>Well, I'm not sure about that... As we already stated, it would make our CI tests crash. And also, Scott has shown that it can be easily overlooked and make the user conclude that the tutorial is bugged.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Changed in f33eb885 for the phrasing

Reply:
<p>I already mentioned my feeling about the tooltips. Here, linking to the group components would do, but for consistency, and because group have not been mentioned yet, I would suggest the same as before, i.e. only categories without any link.</p><p> </p>

Reply:
<p>I agree with Christophe regarding folder links.</p><p> </p>